### PR TITLE
Updating the site with PELUX 1.0

### PR DESCRIPTION
**Pull request by tobsan**

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -16,25 +16,18 @@ well as extensive documentation.
...
- - <del>PDK, SDK and ADK installer</del>
- - Publicly accessible CI
+The PELUX project aims to provide biannual releases. The first release was done
+in January 2018. For already released versions, see the [downloads](/downloads)
+page.
 
 ### PELUX 1.1 [Spring 2018]
-=======
 Content of this release is under discussion. Tentatively the topics in focus will be:
 - Audio manager
```
Comment (GordanM) [inline roadmap.md]: I would remove this line also :).

File: roadmap.md [hunk 2]
```diff
@@ -16,25 +16,18 @@ well as extensive documentation.
...
 - Device manager
```
Comment (GordanM) [inline roadmap.md]: ... and this one. Because there is quite some uncertainty.

File: roadmap.md [hunk 3]
```diff
@@ -16,25 +16,18 @@ well as extensive documentation.
...
-- ARP Board support
 - Experimental support for QEMU
 - Experimental support for TX 2
+
+### PELUX 2.0 [Later 2018]
+- ARP Board support
+ - Application Development Kit (ADK)
```
Comment (GordanM) [inline roadmap.md]: I think there is an extra space here, as well as in the line below.

File: download.md [hunk 4]
```diff
@@ -4,9 +4,66 @@ title: Downloads
 permalink: /downloads/
 ---
 
-Once the first version of PELUX is released you will be able to find download
-links to the images, SDK and ADK, etc.
+# PELUX 1.0
```
Comment (GordanM) [inline download.md]: Maybe there should be a place with link to the SWF (published docs not sources) with that version in the bottom or somewhere?

File: download.md [hunk 5]
```diff
@@ -4,9 +4,66 @@ title: Downloads
...
+
+## Metadata
+Version number: 1.0
+Yocto release: morty and pyro
+Qt version: 5.9.3
+
+## Changelog
+This is the initial release of PELUX.
+
+## Known issues
```
Comment (GordanM) [inline download.md]: Could you make this in to a bullet list? We should have open github issues we can assign to these ideally or if applicable. I can do that later on.

Comment (tobsan) [reply download.md]: It is a bullet list :)

**Pull request merged**